### PR TITLE
[iOS] Fixed potential native animation not terminating

### PR DESCRIPTION
**Pull request by carldebilly**
GitHub Issue (If applicable): closes https://github.com/unoplatform/nventive-private/issues/272

# Bugfix
Fixed a problematic case where a native animation can be _lost_ if the GC is run during the duration of the animation, preventing the animation from being removed and leaving the UI in an undetermined undesired state.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [X] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [X] Contains **NO** breaking changes
- [X] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [X] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

### CODE REVIEW

File: src/Uno.UI.RuntimeTests/Uno.UI.RuntimeTests.csproj [hunk 1]
```diff
@@ -112,11 +112,22 @@
 		</ProjectReference>
 	</ItemGroup>
 	
+	<ItemGroup>
+	  <None Remove="Tests\Windows_UI_Xaml\Controls\Animation_Leak.xaml" />
```
Comment (davidjohnoliver) [inline src/Uno.UI.RuntimeTests/Uno.UI.RuntimeTests.csproj:116]: revert these changes

**Review by davidjohnoliver — APPROVED** [review]

**Review by jeromelaban — APPROVED** [review]
